rest-openehr/pom.xml

```diff
@@ -74,5 +74,9 @@
             <artifactId>ipf-atna-spring-boot-starter</artifactId>
             <version>4.0.0</version>
         </dependency>
+        <dependency>
+            <groupId>org.springframework.security</groupId>
+            <artifactId>spring-security-core</artifactId>
+        </dependency>
```

Comment:
I'm not sure... I think it sneaked in due to some dependency caching issue I experienced with IntellIJ. I will checked it again tomorrow, it might have disappeared in the meantime,